introduction.md

```diff
@@ -1,2 +1,4 @@
 Hi i'm doing Mastering HITHUB!
 My favorite path in Code School is Javascript.
+*Git, javascript, PHP, C++, C#, Android, videogames
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?